lib/storage/bucket.js

```diff
@@ -267,6 +287,9 @@ Bucket.prototype.getFiles = function(query, callback) {
       file.metadata = item;
       return file;
     });
+    if (resp.prefixes) {
+      files.prefixes = resp.prefixes; // bad idea?
```

Comment:
HAHA that's what I'm trying to figure out too. It's a returned value from the API that we're trying to figure what it actually does and how important it is. This is the simplest way I could think of returning them without breaking too much or complicating the API beyond what's necessary.